vllm/v1/core/kv_cache_manager.py

```diff
@@ -39,8 +39,8 @@ def __init__(
 
         self.enable_caching = enable_caching
         self.caching_hash_fn = sha256 if caching_hash_algo == "sha256" else hash
-        # FIXME: make prefix cache stats conditional on log_stats
```

Comment:
We still need this comment because when the log stats is enabled there are still potential configs we could expose in the future.